KBaseGenomeAnnotations/KBaseGenomeAnnotations.spec

```diff
@@ -546,14 +580,6 @@ typedef structure {
 } GenomeAnnotation; 
 
 
-
```

Comment:
Already declared above. Little worried that apparently this got past the workspace type checker... it's severely out of date